### PR TITLE
Keywords in completion

### LINKED ISSUE #347: Completions are missing `do`
When starting a file, and writing `main = do`, I get completions for `drop` etc. I suggest that keywords should be available as completions always. Particularly problematic because after `do` I often want a newline, but in this context enter inserts the word `drop`.

image

### PR DESCRIPTION
**Pull request by serras**
Fixes #347

### CODE REVIEW

File: src/Development/IDE/Core/Completions.hs [hunk 1]
```diff
@@ -361,6 +361,7 @@ getCompletions ideOpts CC { allModNamesAsNS, unqualCompls, qualCompls, importabl
       filtImportCompls = filtListWith (mkImportCompl enteredQual) importableModules
       filtPragmaCompls = filtListWithSnippet mkPragmaCompl validPragmas
       filtOptsCompls   = filtListWith mkExtCompl
+      filtKeywrdCompls = if T.null prefixModule then filtListWith mkExtCompl keywords else []
```
Comment (fendor) [inline src/Development/IDE/Core/Completions.hs]:
```suggestion
      filtKeywordCompls = if T.null prefixModule then filtListWith mkExtCompl keywords else []
```
Looks like a typo? Or is that intentional to match length of the other compls?

Comment (serras) [reply src/Development/IDE/Core/Completions.hs]: The latter 🙄

Comment (cocreature) [reply src/Development/IDE/Core/Completions.hs]: Just don’t align things. It only makes diffs noisy and introducing typo-like things to keep alignment is confusing.

Comment (serras) [reply src/Development/IDE/Core/Completions.hs]: 👍 I've commited the suggestion

Comment (fendor) [reply src/Development/IDE/Core/Completions.hs]: I am sorry, but the suggestion is probably not enough, arent you using the function in another location?

File: src/Development/IDE/Core/Completions.hs [hunk 2]
```diff
@@ -380,6 +381,7 @@ getCompletions ideOpts CC { allModNamesAsNS, unqualCompls, qualCompls, importabl
         | otherwise
         = filtModNameCompls ++ map (toggleSnippets caps withSnippets
                                       . mkCompl ideOpts . stripAutoGenerated) filtCompls
+                            ++ filtKeywrdCompls
```
Comment (fendor) [inline src/Development/IDE/Core/Completions.hs]:
```suggestion
                            ++ filtKeywordCompls
```

Comment (fendor) [reply src/Development/IDE/Core/Completions.hs]: Sorry, too slow.

**Pull request merged**